### ISSUE TITLE
VariantManagement - dropdown button name/label not read by screenreader

### ISSUE THREAD

**Issue by I506645** [SAP/ui5-webcomponents-react#4440]
### Describe the bug

The dropdown/arrow-down button (Select view) of Variant Management to view the saved variants doesn't have a defined name/label or tooltip. This is causing an issue with accessibility. The screen reader is not reading the button name/label/tooltip.
There's no such property for VariantManagement component through which it could be resolved and there's nothing could be done by the application team to resolve this.

The product standard with accessibility is that every user interface should have a defined name and same should be read with corresponding label by screen reader.

![image](https://user-images.githubusercontent.com/121158363/229062168-f6a721a3-ed10-4abd-8832-5167068d1959.png)

### Reproduction steps

Use any screen reader to view the saved variants via the dropdown button and observe that the screen reader reads the button as just 'button'.

### Expected Behaviour

The expected behavior is that the dropdown button should have a tooltip or label that could be defined by the application team or the framework provides a default label for that button. This would then be read by the screen reader.

### UI5 Web Components for React Version

0.27.5

### UI5 Web Components Version

1.6.0

### Browser

Chrome, Edge

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by MarcusNotheis** [comment]
Hi @I506645,

the missing accessibility attribute for the VariantManagement Button has already been reported in #3123 and was fixed in our [1.4.0 release](https://github.com/SAP/ui5-webcomponents-react/releases/tag/v1.4.0).

Can you please try to upgrade your app to a more recent (preferably latest) version of UI5 Web Components (React) and check whether the issue is resolved then?

**Comment by I506645** [comment]
Hi,

Thanks for the reply. We'll upgrade our React web-components version and check this.

Best Regards,
Raghav
